tap_suiteql/streams.py

```diff
@@ -92,3 +92,24 @@ class SubscriptionPriceInterval(suiteqlStream):
         th.Property("status", th.StringType),
         th.Property("subscription", th.StringType),
     ).to_dict()
+
+class ChangeOrderLineStream(suiteqlStream):
+    name = "ChangeOrderLine"
+    path = "/query/v1/suiteql"
+    # Always sort the replication key and format the replication_key
```

Comment:
this comment can be removed because this query does not have a replication key

Reply:
Missed it! Thanks !